### PR TITLE
Enable containerization 

### PR DESCRIPTION
**Pull request by am357**
*Issue #, if available:*

#166 

*Description of changes:*

This PR cannot get merged before #177 

This PR includes the following:

- Adds the required docker config and build scripts to enable containerizing the
  PartiQL playground.

TODO:
- [x] Node.js server listener
- [x] Remove `ace-builds` submodule.
- [x] Containerization
- [ ] REST API
- [ ] Client side URL export
- [ ] AST Graph pan and zooming bugfix

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: partiql-playground/README.md [hunk 1]
```diff
@@ -37,6 +37,39 @@ node src/server.ts
...
+```
+2. Build the package:
+```bash
+make build
+```
+3. Build the container:
+```bash
+make container-build
+```
+3. Run the container:
```
Comment (alancai98) [inline partiql-playground/README.md:56]: Numbering is off here

Comment (am357) [reply partiql-playground/README.md:56]: Yep, I'd exclude the `README.md` review in this PR since it'll get overwritten in the 3rd PR.

Comment (am357) [reply partiql-playground/README.md:56]: Actually, that's not the case for the numbers; let me fix those.

Comment (am357) [reply partiql-playground/README.md]: Fixed.

**Review by alancai98 — APPROVED** [review]

**Pull request merged**